tests/test_variables.py

```diff
@@ -1673,104 +1660,6 @@ def test_get_auto_install_variables_cronjob_id(self):
             )
         )
 
-    def test_get_remove_do_api_tokens(self):
-        config.get_value.cache_clear()
-        feature_flags.get_value.cache_clear()
-        auth.get_tokens.cache_clear()
-
-        _, self.api_tokens_filename = tempfile.mkstemp(suffix=".yml")
```

Comment:
this test has been deleted because we have removed the ability to configure the location of api_tokens via a config file.

this will be centrally managed and, in a later PR, there will be support for mulitple of these files